tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -52,3 +68,48 @@ def dismiss_tooltip(self):
         confirm_element = Wait(self.marionette).until(expected.element_present(*self._button_popup_intro))
         Wait(self.marionette).until(expected.element_displayed(confirm_element))
         confirm_element.tap()
+
+
+class BugzillaLiteStage(BugzillaLite):
+
+
+    name =  'Bugzilla Lite Stage'
+    manifest_url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
+
+    def dismiss_content(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._content_locator))))
+        self.marionette.find_element(*self._close_locator).tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._login_form_locator))
+        import time
+        time.sleep(1)
+
+    def create_new_bug(self, title, description):
+        self.marionette.find_element(*self._create_button).tap()
+        from gaiatest.apps.bzlite.regions.create_bug import CreateBug
+        
+        CreateBug(self.marionette)._fill_title(title)
+        CreateBug(self.marionette)._fill_description(description)
+        CreateBug(self.marionette)._fill_picture()
+        CreateBug(self.marionette)._submit()
+
+        time.sleep(10)
+        Wait(self.marionette,timeout=20).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._bugpage_locator))))
+
+    def navigate_filed_bug(self):
+        self.marionette.find_element(*self._back_locator).tap()
+        self.marionette.find_element(*self._filed_bug_locator).tap()
+        self.marionette.find_element(*self._filed_bug_name).tap()
+       
+        time.sleep(2)
+        details_element = self.marionette.find_element(*self._detailsLink_locator)
+        details_element.tap()
+        time.sleep(2)
+        attach_element = self.marionette.find_element(*self._attachLink_locator)
+        attach_element.tap()
+        time.sleep(2)
+        comment_element = self.marionette.find_element(*self._commentsLink_locator)
+        comment_element.tap()
```

Comment:
Having some trouble to name a function is a sign that this something may need some changes :)

The things we have to do is to check that these tabs show the correct information, thanks to assertions. Assertions have to be in the test itself, so you this function will have to be broken down. So the test will look like this:
``` py
bug_comments = bug_detais.open_comments()
self.assertEqual(bug_comments.bug_title, bug_title)
# do more assertions about the summary and the picture
bug_details = bug_details.open_details()
self.assertEqual(bug_details.bug_title, bug_title)
# do more assertions about the summary, the date filed, the author

# and same thing with the attachments
```